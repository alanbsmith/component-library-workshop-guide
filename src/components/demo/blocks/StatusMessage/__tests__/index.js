import { generateSnapshots } from "__tests__/helpers";

import StatusMessage from '../index';

generateSnapshots(StatusMessage);
