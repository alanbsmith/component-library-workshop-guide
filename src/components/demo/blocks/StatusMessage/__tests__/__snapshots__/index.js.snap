// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #9aa0ac;
}

<div
  className="c0"
/>
`;

exports[`Row should apply statusColorDanger modifier 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #9aa0ac;
  color: #DD3800;
}

<div
  className="c0"
/>
`;

exports[`Row should apply statusColorInfo modifier 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #9aa0ac;
  color: #0975E0;
}

<div
  className="c0"
/>
`;

exports[`Row should apply statusColorSuccess modifier 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #9aa0ac;
  color: #008900;
}

<div
  className="c0"
/>
`;

exports[`Row should apply statusColorWarning modifier 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #9aa0ac;
  color: #967100;
}

<div
  className="c0"
/>
`;
